hw/chip/mimxrt10xx/mimxrt101x/pinning_options.h

```diff
@@ -0,0 +1,141 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020, Dave Marples
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+#ifndef _PINNING_OPTIONS_H_
+#define _PINNING_OPTIONS_H_
+
+// UART Options =====================================
+
+#if defined(PINNING_UART_LPUART1_GPIO_09_10)
+
+#define UART_PORT       LPUART1
+#define PIN_UART_RX     IOMUXC_GPIO_09_LPUART1_RXD
+#define PIN_UART_RX_MUX 0U
+#define PIN_UART_RX_CFG 0x10B0u
+#define PIN_UART_TX     IOMUXC_GPIO_10_LPUART1_TXD
+#define PIN_UART_TX_MUX 0U
+#define PIN_UART_TX_CFG 0x10B0u
```

Comment:
Don't think this is the right way to go, think how many different UARTs there are and how many pin configs each can have, depending on the number of boards we could have double-digit amount of defines here. 

The mux and cfg values will always be the same, so we only need the port and PIN_UART_RX and PIN_UART_TX, these can be moved back to the board_config.h

Reply:
You were the one wanting to clean up board_config.h :-)  
OK, moved them back and left the MUX and CFG values here, ifdef guarded in case someone wants to override them.